Sources/CNIOWindows/include/module.modulemap

```diff
@@ -0,0 +1,4 @@
+module CNIOWindows {
+  header "CNIOWindows.h"
+  export *
+}
```

Comment:
Is the module map required? We shouldn't need it for SwiftPM, but I'm acutely aware that the CMake build may have other requirements.

Reply:
It is needed for the cmake based build.  I’ve had it locally for some time.

Reply:
Ok, I think this shouldn't cause a problem with `generate-xcodeproj` because we shouldn't be importing this on any platform where it matters. /cc @weissi to get his input.

Reply:
Hmm, I'm kind of scared we break this again: https://bugs.swift.org/browse/SR-9024

Reply:
We need to test that the following setup works:

- creating an Xcode project with NIO with this
- creating a new Xcode project for an app
- then dragging the generated NIO Xcode project as a sub-project into the app Xcode project
- make sure everything still works

So far, this always broke as soon as a modulemap was added anywhere.

Reply:
Who's volunteering to do that screwing around? :wink:

Reply:
As soon as theres an Xcode on Windows :-p

Reply:
I think we still need to validate this. I'll do so today.

Reply:
No observed problem on Xcode 11.4.

Reply:
Thanks @Lukasa for testing it